src/gei/Commands/MigrateRepoCommand.cs

```diff
@@ -47,6 +47,21 @@ public MigrateRepoCommand(OctoLogger log, ITargetGithubApiFactory targetGithubAp
                 IsRequired = false,
                 Description = "Defaults to the name of source-repo"
             };
+            var targetApiUrl = new Option<string>("--target-api-url")
+            {
+                IsRequired = false,
+                Description = "The URL of the target API. Defaults to https://api.github.com"
+            };
+            var metadataArchiveUrl = new Option<string>("--metadata-archive-url")
+            {
+                IsRequired = false,
+                Description = "An authenticated SAS URL to an Azure Blob Storage container with the metadata archive. Must be passed in when also using --git-archive-url"
```

Comment:
I think we should add a bit more here. I can see customers seeing this new arg and being confused about whether they need to use it or not. In what circumstances do customers need to provide these args when calling migrate-repo? We should answer that question in the description.

Reply:
Updated in https://github.com/github/gh-gei/pull/221/commits/ab4d0fa838c0d4ed6eea15245a690726f87b1527